### ISSUE TITLE
Is there a way to parse and save default experiment data from the XML in the modelDescription struct?

### ISSUE THREAD

**Issue by sharanry** [ThummeTo/FMI.jl#74]
@ThummeTo 

CC: @ChrisRackauckas

**Comment by ThummeTo** [comment]
Not yet, but its pretty easy to add. I will do so in the next update!

We will parse the value as part of the `modelDescription`-struct and use it as default value for `fmiSetupExperiment` (instead of the current values `0.0` and `1.0` if this information is part of the model description.

Sounds good @sharanry ?
Best regards!

**Comment by sharanry** [comment]
Much appreciated!

Also, `tolerance` is also often available as part of the default experiment metadata.

**Comment by ThummeTo** [comment]
Fixed in 0.3.8 (coming soon).
The values from the ModelDescription are the deafult values for `fmiSimulate` (and other simulation commands).

You can access the values via:
```
fmu.modelDescription.defaultStartTime       
fmu.modelDescription.defaultStopTime                        
fmu.modelDescription.defaultTolerance             
```
